setup.py

```diff
@@ -22,7 +22,7 @@
       ],
       extras_require={
           'dev': [
-              'pylint== 2.6.*',
+              'pylint>= 2.6,< 2.10',
```

Comment:
can we have a deterministic version? like pin it to max version in this range, `2.9.*`

Reply:
addressed by 6a6a58c825790f818cc8186ad072027f2ee517b7